using-mobile-extensions/places-extension-1/places-monitoring-extension/places-monitor-api-reference.md

```diff
@@ -59,35 +59,62 @@ Returns the current version of the `ACPPlacesMonitor` extension.
 
 ## Start monitoring
 
-This API tracks the device's location and monitors the POIs that are near by. You can use this method to start monitoring the user's GPS location. Location monitoring can be done in one of the following ways:
+This API starts tracking the device's location and monitors their nearby Places
 
-* **Continuous**
+When called, the Places Monitor will do the following:
 
-  The monitoring extension receives and processes locations more frequently. This monitoring strategy consumes a lot of power but provides higher accuracy. You can enable continuous monitoring by passing `YES` to the `continuousMonitoring` parameter. For more information, see [Apple documentation on continuous monitoring](https://developer.apple.com/documentation/corelocation/cllocationmanager/1423750-startupdatinglocation).
-
-* **Significant**
-
-  The monitoring extension only receives and processes location updates after the device has moved a significant distance from the previously processed location. This monitoring strategy consumes less power than the continuous monitoring strategy. You can enable significant monitoring by passing `NO` to the `continuousMonitoring` parameter. For more information, see [Apple documentation on significant monitoring](https://developer.apple.com/documentation/corelocation/cllocationmanager/1423531-startmonitoringsignificantlocati)
+- If the authorization for the location service has not been provided for the application, the first call to the `start` API requests for authorization to use the location service as configured for the application
```

Comment:
- Please add a period to the end of this sentence.

- Please replace "for" with "the", so that it reads as follows:

"If the authorization for the location service has not been provided for the application, the first call to the `start` API requests the authorization to use the location service as configured for the application."